sdk/Trace/Span.php

```diff
@@ -16,10 +16,8 @@ class Span implements API\Span
     private $startEpochTimestamp;
     private $start;
     private $end;
-    private $statusCode = API\SpanStatus::OK;
```

Comment:
These attributes are initialized in the constructor, so there is no need to do it here.